Description: Suppress the shutdown option in the panel if LTSP_CLIENT is set
Bug: https://bugs.launchpad.net/ubuntu/+source/gnome-panel/+bug/282669

Index: gnome-panel-2.30.0/gnome-panel/panel-menu-items.c
===================================================================
--- gnome-panel-2.30.0.orig/gnome-panel/panel-menu-items.c	2010-06-03 13:58:17.316446305 +1000
+++ gnome-panel-2.30.0/gnome-panel/panel-menu-items.c	2010-06-03 13:58:18.496449211 +1000
@@ -1706,7 +1706,7 @@
 	}
 
 	item = panel_menu_items_create_action_item (PANEL_ACTION_SHUTDOWN);
-	if (item != NULL) {
+	if (item != NULL && !g_getenv("LTSP_CLIENT")){
 		if (!separator_inserted)
 			separator = add_menu_separator (menu);
 
